File: Łatwe/Java/StringMerge/StringMerge.java
import java.util.Scanner;

class StringMerge {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i = 0; i < t; i++) {
            String firstString = in.next();
            String secondString = in.next();
            System.out.println(string_merge(firstString, secondString));
        }
    }

    public static String string_merge(String firstString, String secondString) {
        String mergedString = "";
        if (firstString.length() > secondString.length()) {
            for (int i = 0; i < secondString.length(); i++)
                mergedString += firstString.charAt(i) + "" + secondString.charAt(i);
        } else {
            for (int i = 0; i < firstString.length(); i++)
                mergedString += firstString.charAt(i) + "" + secondString.charAt(i);
        }
        return mergedString;
    }
}

File: Łatwe/C++/StringMerge/StringMerge.cpp
#include <bits/stdc++.h>

char* string_merge(char* firstString, char* secondString);

int main() {
    int t;
    std::cin >> t;
    char* firstString = new char[100];
    char* secondString = new char[100];
    for (int i = 0; i < t; i++) {
        std::cin >> firstString >> secondString;
        char* answer = string_merge(firstString, secondString);
        std::cout << answer << std::endl;
        delete [] answer;
    }
}

char* string_merge(char* firstString, char* secondString) {
    char* mergedString;
    std::string temp;
    if (strlen(firstString) > strlen(secondString)) {
        mergedString = new char[strlen(secondString) * 2 + 1];
        for (int i = 0; i < strlen(secondString); i++) {
            temp += firstString[i];
            temp += secondString[i];
        }
    } else {
        mergedString = new char[strlen(firstString) * 2 + 1];
        for (int i = 0; i < strlen(firstString); i++) {
            temp += firstString[i];
            temp += secondString[i];
        }
    }
    strcpy(mergedString, temp.c_str());
    return mergedString;
}
